spec/runtime/setup_spec.rb

```diff
@@ -1213,7 +1213,7 @@ def lock_with(ruby_version = nil)
 
     describe "default gem activation" do
       let(:exemptions) do
-        if Gem::Version.new(Gem::VERSION) >= Gem::Version.new("2.7") || ENV["RGV"] == "master"
```

Comment:
why we don't need this check anymore?

Reply:
Rubygems master is 3.1 nowadays, so the first condition should suffice, I think.

But, yeah, this change is actually independent from this PR, but since I was at it I reviewed usages of the `RGV` environment variable and found this one was unnecessary.